fdbclient/MasterProxyInterface.h

```diff
@@ -49,9 +49,9 @@ struct MasterProxyInterface {
 
 	RequestStream< struct GetRawCommittedVersionRequest > getRawCommittedVersion;
 	RequestStream< struct TxnStateRequest >  txnState;
-	RequestStream<struct ExecRequest> execReq;
-
 	RequestStream< struct GetHealthMetricsRequest > getHealthMetrics;
+	RequestStream<struct ExecRequest> execReq;
+	RequestStream<struct ProxySnapRequest> proxySnapReq;
```

Comment:
Can you please add a comment here explaining what the difference is between these two?

Reply:
V1 is going away in a short time even before 6.2, i will take the liberty not to address it.